cwr/convert.go

```diff
@@ -42,31 +57,206 @@ func NewConverter(store *meta.Store) *Converter {
 	}
 }
 
-// ConvertRegisteredWork reads all registeredWork , converts them to META
-// objects, stores them in the META store and sends their CIDs to the given
-// stream.
-func (c *Converter) ConvertRegisteredWork(ctx context.Context, outStream chan *cid.Cid, cwrFileReader io.Reader) error {
-	// get records from the db
-	records, err := ParseCWRFile(cwrFileReader)
+// Transaction struct
```

Comment:
This comment is not very informative to the point where it may as well not be there.

How about:

```
Transaction represents a CWR transaction which is either an
NWR, REV, EXC, ACK, AGR or ISW record.
```